src/vertex_array_object.rs

```diff
@@ -520,12 +544,24 @@ unsafe fn bind_attribute(ctxt: &mut CommandContext<'_>, program: &Program,
     }
 
     // binding attributes
-    for &(ref name, offset, ty, normalize) in bindings.iter() {
+    for &(ref name, offset, location, ty, normalize) in bindings.iter() {
         let (data_type, elements_count, instances_count) = vertex_binding_type_to_gl(ty);
 
-        let attribute = match program.get_attribute(Borrow::<str>::borrow(name)) {
-            Some(a) => a,
-            None => continue
+        let attribute = match location {
+            -1 => {
+                // No location precised in Vertex Format. Check name instead
```

Comment:
I don't think "precised" is a valid english word.

Reply:
you are right